actions/hosts.last_checkin.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: hosts.last_checkin
+pack: foreman
+description: List all hosts that haven't reported back 
+runner_type: orquesta
+entry_point: workflows/hosts.last_checkin.yaml
+enabled: true
+parameters:
+  conection:
+    type: string
+    description: "Name of <connection> from this pack's configuration that specifies how to connect to a server."
+    required: false
+  server:
+    required: true
+    type: string
+  username: 
+    required: true
+    type: string
+  password:
+    required: true
+    type: string
+    secret: true
+  time:
+    required: true
+    type: string
+  search:
+    type: string
+    default: "last_checkin < "
+  per_page:
+    type: string
+    default: "25"
```

Comment:
this should be a large number because the foreman library under the hood doesn't handle paging properly, so if this is set to 25, we'll only get 25 results max

also, shouldn't the type be `integer` and not a string?